pkg/shell/nav.scss

```diff
@@ -462,6 +462,10 @@ $desktop: $phone + 1px;
         &:focus {
             text-decoration: none;
         }
+
+        &:focus {
```

Comment:
Thanks, just wonder if there was a good reason why it is not in the same rule above, but a separate one? (Now we have `&:focus` twice)

Reply:
...because the rule above isn't just &:focus, but also &:hover? We're not changing the outline for hover, but for focus only. (It wouldn't hurt to change it for hover, but I wanted the CSS to be explicit that it's changing the focus ring offset.)

Reply:
It seems GitHub compressed the non-changed lines. Just for clarification, the existing block is:

```scss
        &:hover,
        &:focus {
            text-decoration: none;
        }
```

Reply:
oh, of course! Sorry, totally missed that